platform/node/cmake/module.windows.cmake

```diff
@@ -0,0 +1,239 @@
+# We need CMake 3.9 because the Xcode generator doesn't know about systems include paths before 3.9.
+# See https://gitlab.kitware.com/cmake/cmake/issues/16795
+cmake_minimum_required(VERSION 3.9)
+
+if (NOT NODE_MODULE_MINIMUM_ABI)
```

Comment:
Why do you need so many Node specifics for Windows but not for other platforms?

Reply:
I asked about this file, and the linux/macos equivalent is in the `mapbox/cmake-node-module` package
https://github.com/mapbox/cmake-node-module/blob/master/module.cmake

We talked a bit about if it may make sense to move the one for other platforms out of the cmake-node-module package so we have some control over it.

Reply:
OK, I'm fine with including this for now then.

Reply:
Just a side fact. The linux version of the file being in a node package actually gave me some trouble in my cmake-js PR, because the path of node_modules changes if maplibre-native is used as a submodule

https://github.com/maplibre/maplibre-gl-native/pull/535/files#diff-6d48556e62e5d1ff41c30848ca089f0bb2fe15ddf3b58bcc1f33e902b7022eddR1-R6

Reply:
@tdcosta100 just wondering, would this new file also be compatible with the linux/macos builds. like maybe replace module.cmake with this new file all together?

Looking at the difference it seems like mostly a few path changes and sections for WIN32 added.